app/components/mentor-card.js

```diff
@@ -0,0 +1,4 @@
+import Ember from 'ember';
```

Comment:
You don't need to create a js file if you're not making any changes to the component's JS. Pls, remove this file